### DISCUSSION TITLE
Bug or ?

### DISCUSSION THREAD

**Discussion by sec13b** [community/community#137321] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I ask on : https://github.com/0xthirteen/SharpRDP/issues/18

But someone come fast (2seconds) with a "solution"

![image](https://github.com/user-attachments/assets/fe1f69eb-2817-47da-a3db-603b2fde9fc3)

![image](https://github.com/user-attachments/assets/0f1b5a22-831e-4a40-bc8b-25465b41b561)

**Comment by joemar25** [comment]
Hi @sec13b ,

It looks like you might be dealing with a similar situation that I've seen in another GitHub issue. The “solution” posted within seconds by the user "KrissV2" seems suspiciously fast and involves a link to download a compressed file from MediaFire with the password 'changeme'. This is almost identical to another case where the same user or a similar spammer posted a link in another issue thread.

**Here’s what I’d recommend:**

1. **Do not click or download the file**: The file could contain malware or be part of a phishing scam.
   
2. **Report the user and the comment**: Use GitHub's "Report abuse" feature to alert the moderators about this behavior. It looks like a spam or malicious attempt.

3. **Block the user**: If you manage the repository, block this user to prevent them from posting more of these comments.

4. **Delete the comment**: Removing the comment helps keep your repository clean and safe for other users who might not recognize it as spam.

5. **Be Cautious**: Make sure not to engage with such comments and encourage others in your project or community to do the same.

It’s always better to be cautious, especially with these types of fast “solutions” that involve downloading files from third-party sites.

Stay safe!

**Comment by sec13b** [comment]
its not first time , when i see this , i see already github delete

**Reply by joemar25** [reply]
You can delete it then sir. It is bad to have something like that in your account.

**Comment by sec13b** [comment]
i dont think come from my account .
